server/admin_management/api/app/routers/router_guards.py

```diff
@@ -39,7 +39,7 @@
 
 
 def authorize_by_fam_admin(claims: dict = Depends(validate_token)):
-    required_role = "FAM_ACCESS_ADMIN"
+    required_role = "FAM_ADMIN"
```

Comment:
I need to update hardcoded required role in router guard for both backend, and admin_management, because once we merge this pull request, all FAM roles will be added as "{app_name}_ADMIN" instead of "{app_name}_ACCESS_ADMIN"